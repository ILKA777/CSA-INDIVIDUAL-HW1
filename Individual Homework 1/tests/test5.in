20
10 -9 8 123 0 333 -323 -10 100 -34 10 -9 8 123 0 333 -323 -10 100 -34
