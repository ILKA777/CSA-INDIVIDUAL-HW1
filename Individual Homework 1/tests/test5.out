1148-7521148-7521148-7521148-7521148-7521148-7521148-7521148-7521148-7521148-752
